File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/bindings/MyCustomRow.cpp
#include "MyCustomRow.h"
#include <string>

namespace mybindings
{
  std::string MyCustomRow::GetRowText()
  {
    return this->RowText;
  }
  int MyCustomRow::getHandle()
  {
    return this->handle;
  }
  int MyCustomRow::getLevel()
  {
    return this->level;
  }
}

File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/bindings/MyCustomRow.cs
/// <filename>
///     MyCustomRow.cs
/// </filename>


namespace bindings
{
    internal class MyCustomRow
  {
        private string RowText;
     
        public     virtual string GetRowText()
    {
      return this.RowText;
    }
     
        private int handle;
     
        public     virtual int getHandle()
    {
      return this.handle;
    }
     
        private int level;
     
        public     virtual int getLevel()
    {
      return this.level;
    }
  }
}

